docs/architecture/cloud-native/introduction.md

```diff
@@ -1,26 +1,76 @@
 ---
-title: Introduction to Cloud Native .NET Applications
-description: Architecting Cloud Native .NET Apps for Azure | Introduction to Cloud Native .NET Applications
-ms.date: 06/30/2019
+title: Introduction to cloud-native applications
+description: Learn about cloud native computing
+author: robvet
+ms.date: 08/20/2019
 ---
-# Introduction to Cloud Native .NET Applications
+# Introduction to cloud-native applications
 
-Welcome to the world of Cloud Native computing!
+Another day, at the office, working on the next significant thing.
 
-You first thought might be, “What exactly does that mean?” Another industry buzzword concocted by cloud providers and software vendors to sell us more stuff?”
+Your cellphone rings. It's your friendly recruiter - the one who calls your twice a day about new jobs.
```

Comment:
Change: Your cellphone rings. It's your friendly recruiter - the one who calls ~your~ you twice a day about new jobs.